src/commons_feval_feface.jl

```diff
@@ -0,0 +1,293 @@
+# Common methods for both FEValues and FEFaceValues
+typealias FEValuesOrFEFaceValues{dim,T,FS,GS} Union{FEValues{dim,T,FS,GS}, FEFaceValues{dim,T,FS,GS}}
```

Comment:
AbstractFEValues? It is not important since it is not user facing. Your names are also good. Deprecate fevalues then?